src/main/java/ZeusBot.java

```diff
@@ -0,0 +1,200 @@
+package main.java;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import static java.lang.Integer.parseInt;
+
+public class ZeusBot {
+
+	public static final String TAB = "\t____________________________________________________________";
+	public static final String MSG_EMPTY_INPUT = "\tOops! You've gotta input something~";
+	public static final String MSG_BYE = "\tAwh... so fast? Alright then, hope to see you again soon!";
+	public static final String MSG_DUPLICATE_MARK = "\tYou've already finished this silly!";
+	public static final String MSG_DUPLICATE_UNMARK = "\tAh! That's amazing, you've got another one crossed out~";
+
+	public static void main(String[] args) {
+		ZeusBot.greetUser();
+
+		Scanner sc = new Scanner(System.in);
+		String echo_word = "";
+		ArrayList<Task> todo_list = new ArrayList<>(); //array with class type
+
+		while (true) {
+			echo_word = sc.nextLine();
+			if (echo_word.equals("bye")) {
+				break;
+			} else if (echo_word.isEmpty()) {
+				ZeusBot.warnEmptyInput(); //check empty input
+			} else if (echo_word.startsWith("mark") || echo_word.startsWith("unmark")) {
+				ZeusBot.handleMarking(echo_word, todo_list); //ensure num args == 2
+			} else if (echo_word.equals("list")) {
+				ZeusBot.listTasks(todo_list); //list current tasks
+			} else
+				ZeusBot.addTask(echo_word, todo_list); //add task to list
```

Comment:
Adding to the comment above, switch-case might come in handy here to improve readability.

Also just to note, if you intend to stick with your own implementation - line 33 specifically - you should add the curly braces (the else statement).

```suggestion
			} else {
				ZeusBot.addTask(echo_word, todo_list); //add task to list
			}
```